src/pal/pal_solaris.h

```diff
@@ -0,0 +1,33 @@
+#pragma once
+
+#if defined(__sun)
+// clang-format off
```

Comment:
Why is this needed?

Reply:
ah ... clang-format had the tendency to reorder the inclusions.

Reply:
It shouldn't reorder between blocks - isn't the blank line between them enough?

Reply:
ah ... still reordering it.

Reply:
I would sooner you add 
```
#include <strings.h>
```
into the Pal posix header.  That is calling `bzero`, which is coming from `strings.h`.  I guess other platforms have exposed it without requiring the inclusion.

Reply:
I considered this but was not sure all posix systems has this. We can always try.